frontend/app/templates/index.hbs

```diff
@@ -1,142 +1,173 @@
+            <label>Choose your font</label>
+            <ul style="list-style-type:none">
+              <li><input type="radio" name="fontsource" id="custfont">&nbsp;Use Custom font</li>
+              <section id="custom-font" style="display:none">
+                <label for="inputFile">Select from following fonts</label>
+                <div class="btn-group">
+                  <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown"
+                          aria-haspopup="true" aria-expanded="false">
+                    <span class="placeholder2">Select a font</span>
+                    <span class="glyphicon glyphicon-chevron-down"></span>
+                  </button>
+                  <ul class="dropdown-menu">
+                    <li class="font-options" data-item="monospace" style="font-family:'monospace'">monospace</li>
+                    <li class="font-options" data-item="sans-serif" style="font-family:'sans-serif'">sans-serif</li>
+                    <li class="font-options" data-item="sans" style="font-family:'sans'">sans</li>
+                    <li class="font-options" data-item="Courier 10 Pitch" style="font-family:'Courier 10 Pitch'">Courier
+                      10 Pitch
+                    </li>
+                  </ul>
+                </div>
+              </section>
+              <input type="hidden" name="custfont" value="">
+            </ul>
+            <button type="submit" disabled="disabled" class="btn btn-block submit-btn">Generate Badges</button>
+            <button type="button" disabled="disabled" class="btn btn-block btn-warning" id="preview-btn">Preview
+            </button>
+            <div id="preview" class="preview-image hidden">
+              <li class="preview-image-li" id="preview-li-1"></li>
+              <li class="preview-image-li" id="preview-li-2"></li>
+              <li class="preview-image-li" id="preview-li-3"></li>
+              <li class="preview-image-li" id="preview-li-4"></li>
+              <li class="preview-image-li" id="preview-li-5"></li>
             </div>
-        </form>
+          </div>
+        </div>
+      </div>
+    </form>
+    <div id="success" style="visibility: hidden;">
+      <div class="flash-success">Your badges have been created successfully.</div>
+      <div class="text-center">
+        <a id="badge-link" href="http://localhost:5000/static/badges/{{msg}}-badges.pdf" class="btn btn-success" download="{{msg}}-badges.pdf">Download as
+          PDF</a>
+      </div>
     </div>
+  </div>
 </div>
 
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 <script src="{{rootURL}}js/jquery.minicolors.min.js"></script>
 <script src="{{rootURL}}js/script.js"></script>
 <script src="{{rootURL}}js/form.js"></script>
+<script type="text/javascript">
+  $(document).ready(function () {
```

Comment:
Should we open another issue, so that this script can be kept as a separate js file?

Reply:
May be we can reformat the code structure, but for this PR, it is necessary??

Reply:
No, this PR don't requires it, LGTM 👍